packages/@okta/vuepress-theme-prose/mixins/AnchorHistory.vue

```diff
@@ -83,8 +83,8 @@ export default {
 
       const matchingPair = this.anchorsOffset.find(
         pair =>
-          scrollTop >= pair.start - this.paddedHeaderHeight &&
-          (!pair.end || scrollTop < pair.end - this.paddedHeaderHeight),
+          scrollTop >= pair.start - 20 &&
```

Comment:
`this.paddedHeaderHeight` quals 87 plus another positive number. How does it work when we replace 100+ number with 20?

Reply:
That seems to be the same change as we have in MobileOnThisPage.vue. It compensates `-20` in `scrollToAnchor` function, which should not be needed if `offsetHeight` property worked :( 
let's do some cleanup:
- please remove `paddedHeaderHeight` and the other code that is not used anymore
- move 20 into `LAYOUT_CONSTANTS` - that is basically ANCHOR_TOP_MARGIN. btw can we use 32 so it will match margin on the all pages and will be just a one pixel off on the errors page?
- please remove this unused function too https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/components/OnThisPage.vue#L103

Reply:
> * please remove `paddedHeaderHeight` and the other code that is not used anymore

Removed `paddedHeaderHeight` and the method to calculate it.

> * move 20 into `LAYOUT_CONSTANTS` - that is basically ANCHOR_TOP_MARGIN. btw can we use 32 so it will match margin on the all pages and will be just a one pixel off on the errors page?

Moved the margin value to a constant and used 32px to match the margin of the element.

> * please remove this unused function too https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/components/OnThisPage.vue#L103

Removed unused code

Reply:
Preview link: https://65dedffaeb4ffeb34c014d9d--reverent-murdock-829d24.netlify.app